core/src/main/resources/db/changelog/changesets/2024_04_12_remove_sitemedia_uploadfilename.yaml

```diff
@@ -0,0 +1,8 @@
+databaseChangeLog:
+  - changeSet:
+      id: "remove_sitemedia_uploadfilename"
+      author: sampath
+      changes:
+        - dropColumn:
+            tableName: site_media
+            columnName: upload_file_name
```

Comment:
nit: set your IDE to auto-include a newline at the end of files -- it's the generally accepted POSIX standard  https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline